client/html/posts_header.tpl

```diff
@@ -28,4 +28,11 @@
             %><a href class='mousetrap button append close'>Stop editing safety</a><%
         %></form><%
     %><% } %><%
+    %><% if (ctx.canBulkDelete) { %><%
+        %><form class='horizontal bulk-edit bulk-edit-delete'><%
+            %><a href class='mousetrap button append open'>Mass delete</a><%
+            %><input class='mousetrap start' type='submit' value='Delete selected posts'/><%
```

Comment:
It would be cool to show the text "Delete {Count} posts", but I couldn't think of a good implementation for that.